digitalpublicgoods/statbus.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "Statbus",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.statbus.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "No",
```

Comment:
This field should be changed to `Yes` with the open source alternatives documented in the next field. Can't approve a DPG with `No` in this field.

Reply:
Resolved in commit c48bc0e